testing/testcafe/tests/editors/validationMessage.ts

```diff
@@ -20,3 +27,55 @@ test('ValidationMessage integrated in editor should not raise any errors when it
     .click(checkBox1.element)
     .expect(true).ok();
 });
+
+fixture`ValidationMessage_TextBox`
+  .page(url(__dirname, '../container.html'));
+
+test('Validation Message position should be correct after change visibility of parent container', async (t) => {
+  const { takeScreenshot, compareResults } = createScreenshotsComparer(t);
+
+  await t
+    .click(Selector(`.${TEXTEDITOR_INPUT_CLASS}`))
+    .pressKey('backspace')
+    .pressKey('enter')
+    .pressKey('tab');
+
+  await t
+    .expect(await takeScreenshot('Textbox_validation_message.png'))
+    .ok()
+    .expect(compareResults.isValid())
+    .ok(compareResults.errorMessages());
+
+  await ClientFunction(() => {
+    (document.querySelector('#container') as HTMLElement).setAttribute('hidden', 'true');
+  })();
+
+  await ClientFunction(() => {
+    document.querySelector('#container')?.removeAttribute('hidden');
```

Comment:
cannot we do this in the previous ClientFunction?

Reply:
I do setAttribute and removeAttribute logic in various clientFunction willfully. It helps me separate important parts of scenario logic separate each other and debug it.